pkg/client/handlers.go

```diff
@@ -269,7 +271,37 @@ func (c *Client) fixForwardedFor(next http.Handler) http.Handler {
 	})
 }
 
+func (c *Client) handleTrigger(r *http.Request) (int, interface{}) {
+	trigger := mux.Vars(r)["trigger"]
+	c.Debugf("Incoming API Trigger: %s", trigger)
+
+	const apiTrigger = "apitrigger"
+
+	switch {
+	case trigger == "cfsync":
+		c.notify.SyncCF(false)
+	case trigger == "services" && c.Config.Services.Disabled:
+		return http.StatusNotImplemented, "services not enabled"
+	case trigger == "services":
+		c.Config.Services.RunAllChecksSendResult(apiTrigger)
+	case trigger == "sessions" && !c.Config.Plex.Configured():
+		return http.StatusNotImplemented, "sessions not enabled"
+	case trigger == "sessions":
+		c.notify.SendPlexSessions(apiTrigger)
+	case trigger == "stuckitems":
+		c.notify.SendFinishedQueueItems(c.notify.BaseURL)
+	case trigger == "dashboard":
+		c.notify.GetState()
+	case trigger == "snapshot":
+		c.notify.SendSnapshot(apiTrigger)
+	default:
+		return http.StatusBadRequest, "unknown trigger '" + trigger + "'"
+	}
```

Comment:
These are the available triggers. All of them send a message into a channel the goes into a go routine. As long as the requests don't come in fast they should be quick. Consecutive requests may be slow as the channel blocks while a collection is happening. Collections happen on intervals too, so intermittently requests could be slow.

Reply:
Seems to be working as expected!